packages/cli/src/commands/generate/sdl/__tests__/__snapshots__/sdl.test.js.snap

```diff
@@ -10,7 +10,7 @@ exports[`in javascript mode creates a multi word sdl file 1`] = `
   }
 
   type Query {
-    userProfiles: [UserProfile!]!
+    userProfiles: [UserProfile!]! @skipAuth
```

Comment:
I think we're doing this the wrong way round. It should be `requireAuth` - secure by default right?

Reply:
Correct - but in order for the e2e tests to run and pass, I chose skipAuth to showcase that auth needs to happen.

This is for discussion - lock in down or present that it is open.

Reply:
For the e2e tests and the items to pass, the generators would set requireAuth and then we'd have to coded every sdl to set skipAuth -- this is the DX issue I noted to you and DP to "see how it feels".

Reply:
I don't believe it would be an issue on e2e because we ship by default with getCurrentUser to return true

Reply:
Ok, I'll check but the current directive didn't actually check for isAuthenticated -- the next step is to move that logs from app to framework so I'll try that and change sdl generators.